utils.js

```diff
@@ -748,12 +823,30 @@ function Signer (password, isTestnet, networks, SLIP44, pubTypes) {
   this.accountIndex = 0
 }
 function TrezorSigner (password, isTestnet, networks, SLIP44, pubTypes) {
+  try{
+    if(!trezorInitialized){
+      console.log('Initializing trezor')  
+      TrezorConnect.init({
+        // connectSrc: 'https://localhost:8088/', // un-comment for testing on your localhost connect server
+        lazyLoad: true, // this param will prevent iframe injection until TrezorConnect.method will be called
+        manifest: {
+          email: 'jsidhu@blockchainfoundry.co',
+          appUrl: 'https://syscoin.org/',
+        }
+      });
+      trezorInitialized = true; //Trezor should be initialized on first run only
+    } 
+  
+  
+} catch (e) {
+    throw new Error('TrezorSigner should be called only from browser context: ' + e);
+}
   this.Signer = new Signer(password, isTestnet, networks, SLIP44, pubTypes)
 
   // try to restore, if it does not succeed then initialize from scratch
-  if (!this.Signer.password || !this.restore(this.Signer.password)) {
-    this.createAccount()
-  }
+  // if (!this.Signer.password || !this.restore(this.Signer.password)) {
```

Comment:
upon site launch shouldn't it ask user to create their first account or does trezor already do this? I think first account is required for sjs to operate